waveform_collection/__init__.py

```diff
@@ -32,4 +32,3 @@ def _formatwarning(message, category, *args, **kwargs):
 
 from .server import gather_waveforms, gather_waveforms_bulk, INFRASOUND_CHANNELS
 from .local import read_local, Smart24
-from .local.common import load_json_file
```

Comment:
Suggest keeping this line so that this generally-useful function is exposed, for uses such as
```python
from waveform_collection import load_json_file

coords = load_json_file('local_infra_coords.json')
```